### PR TITLE
Elamin-week3-practise-2-map exercise

### PR DESCRIPTION
**Pull request by aldeez988**

### CODE REVIEW

File: week-3-practise/level-2/2-map.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+var attendances = [
+  ["Ahmed", 8],
+  ["Clement", 10],
+  ["Elamin", 6],
+  ["Adam", 7],
+  ["Tayoa", 11],
+  ["Nina", 10]
+];
+
+var eligibleStudentNames = attendances.filter(s => s[1] >= 8).map(s => s[0]); // TODO: Complete this line.
```
Comment (40thieves) [inline week-3-practise/level-2/2-map.js]: I think this is the answer? We probably want to remove this

Comment (aldeez988) [reply week-3-practise/level-2/2-map.js]: Thank you  @40thieves for your comments, I have updated my PR and I  removed the answer.

**Comment by aldeez988** [comment]
Really thank you @martynamaron, @40thieves and @nbogie  for this opportunity I really appreciate it.

**Comment by martynamaron** [comment]
> Looks good to me!! :shipit: 🎆 🎉 Thanks @aldeez988!

Could we have it merged please?

**Pull request merged**